auth/authn.go

```diff
@@ -106,13 +102,31 @@ func Init() {
 		DB:         database.DB(),
 		Redirector: auth.Redirector{RedirectBack},
 		UserModel:  User{},
+		UserStorer: BanzaiUserStorer{signingKeyBase32: signingKeyBase32, droneDB: initDroneDatabase()},
+		ViewPaths:  []string{"github.com/banzaicloud/pipeline/views"},
```

Comment:
Should `ViewPaths` come from config and default to `github.com/banzaicloud/pipeline/views` if not defined.